scripts/ebounty.lic

```diff
@@ -997,6 +1020,29 @@ module EBounty
     
     
     
+    end
+  
+    def self.set_eval
+    
+      bounty_eval = "(checkbounty =~ /^You (?:succeeded|have located)|your task is failed/)" # bounty complete
+      
+      if checkbounty =~ /A local divinist has had visions of the child fleeing/
+        bounty_eval += " || (checkbounty =~ /^You have made contact with the child/ && GameObj.npcs.any? { |npc| npc.name =~ /\\bchild\\b/ })"
+      elsif checkbounty =~ /^You are not currently assigned / && !Spell[9003].active?
```

Comment:
&& !Spell[9003].active?
This needs to be removed. It doesn't set the evaluation, because the only time you go out and hunt when you are not on a current bounty is if Spell[9003] is active, otherwise you're grabbing a bounty. So this condition is never met. This then causes bigshot to not have an eval that is possible, so it hunts until you run out of resources or get encumbered instead of heading back in to grab a bounty when it's available.